cli/internal/globby/globby.go

```diff
@@ -36,8 +52,8 @@ func GlobFiles(basePath string, includePatterns []string, excludePatterns []stri
 			return nil
 		}
 
-		if val, _ := doublestar.PathMatch(includePattern, p); val || len(includePatterns) == 0 {
-			result = append(result, p)
+		if val, _ := doublestar.PathMatch(includePattern, path); val || len(includePatterns) == 0 {
```

Comment:
ditto re: error

Reply:
ditto re: deferral

Reply:
(I'm hoping to have an alternative implementation up before you wake up.)